libr/bin/bin.c

```diff
@@ -1243,7 +1243,7 @@ R_API RBuffer *r_bin_package(RBin *bin, const char *type, const char *file, RLis
 		RListIter *iter;
 		ut32 num;
 		ut8 *num8 = (ut8*)&num;
-		RBuffer *buf = r_buf_new_file (file, true);
+		RBuffer *buf = r_buf_new_file (file, O_RDWR | O_CREAT, 0644);
```

Comment:
Wat. Why a buffer should use posix flags to create a rw file? Imho a buffer should work only in memory.

Reply:
Mmmm... It was never like this. You could have a buffer that works on a file. However, before you could just say "true/false" to mean "create new file/do not create new file". Now I have given the possibility to the user to specify the flags.